include/psp2kern/kernel/sysroot.h

```diff
@@ -162,6 +162,31 @@ int ksceSysrootIsUpdateMode(void);
 int ksceSysrootIsUsbEnumWakeup(void);
 int ksceSysrootIsExternalBootMode(void);
 
+typedef SceUID (* SceKernelGetSystemSwVersionFunc)(void);
+
+/**
+ * Set GetSystemSwVersion function.
+ *
+ * @param[in] func - The GetSystemSwVersion function pointer.
+ *
+ * @return none.
+ */
+void ksceKernelSysrootSetGetSystemSwVersionFunc(SceKernelGetSystemSwVersionFunc func);
+
+/**
+ * Get System Sw Version
+ *
+ * @return SystemSwVersion on success, < 0 on error.
+ */
+int ksceKernelSysrootGetSystemSwVersion(void);
```

Comment:
could you update the document about `ksceKernelSysrootSetGetSystemSwVersionFunc`?
it will call function which set via `ksceKernelSysrootSetGetSystemSwVersionFunc`.